lib/norm/conformer.ex

```diff
@@ -43,7 +43,12 @@ defmodule Norm.Conformer do
   defp format_val({:index, i}), do: "[#{i}]"
   defp format_val(t) when is_tuple(t), do: "#{inspect(t)}"
   defp format_val(l) when is_list(l), do: "#{inspect(l)}"
-  defp format_val(msg), do: "#{msg}"
+
+  defp format_val(msg) do
+    "#{msg}"
+  rescue
+    Protocol.UndefinedError -> inspect(msg)
+  end
```

Comment:
instead of relying on exception we could check if protocol is implemented with `if String.Chars.impl_for(msg)`. And instead of only on exception, I think using inspect is a decent general fallback:

```suggestion
  defp format_val(msg), do: inspect(msg)
```